### PR TITLE
feat: razorpay

### PR DESCRIPTION
**Pull request by ArshdeepGrover**
# PR Template

## Type

- ( ) Fix
- ( ) Refactor
- ( ) Style
- ( ) Docs
- (X) Feat
- ( ) Hotfix

### CODE REVIEW

File: apps/commudle-admin/src/app/components/fill-data-form/fill-data-form-paid/fill-data-form-paid.component.html [hunk 1]
```diff
@@ -300,6 +300,12 @@
               </div>
             </nb-card-body>
           </nb-card>
+          <div class="com-my-2 com-px-2" *ngIf="!ticketPaidAlready">
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/components/fill-data-form/fill-data-form-paid/fill-data-form-paid.component.html:303]: Move this css in scss file

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-payments/community-payments.component.html [hunk 2]
```diff
@@ -81,155 +87,648 @@
...
-                <select formControlName="phone_country_code">
-                  <option *ngFor="let country of countries" [value]="country.phone">
-                    +{{ country.phone }} ({{ country.name }})
+        </form>
+      </div>
+      <br />
+      <!-- razorpayAccountForm -->
+      <div *ngIf="selectedCountry === 'India' && !isLoading">
+        <form [formGroup]="razorpayAccountForm">
+          <div class="com-flex com-flex-col com-gap-3">
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-payments/community-payments.component.html]: move this css in scss

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-payments/community-payments.component.html [hunk 3]
```diff
@@ -240,28 +739,181 @@
...
+                "
+              >
+                <commudle-alert [error]="true" [errorMessage]="'Required Field'"></commudle-alert>
+              </span>
+            </label>
+
+            <!-- Phone* -->
+            <label>
+              Phone*
+              <div class="com-flex com-flex-row com-gap-2 com-h-auto">
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-payments/community-payments.component.html]: move this css

File: apps/commudle-admin/src/app/components/fill-data-form/fill-data-form-paid/fill-data-form-paid.component.ts [hunk 4]
```diff
@@ -534,23 +557,77 @@ export class FillDataFormPaidComponent implements OnInit, OnDestroy, AfterViewIn
           });
         } else {
           if (result.paymentIntent.status === 'succeeded') {
+            // this.ticketPaid = true;
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/components/fill-data-form/fill-data-form-paid/fill-data-form-paid.component.ts]: remove comment if not required

File: apps/shared-components/paid-form-list/paid-form-list.component.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  @Input() communityId: number | string;
+  eventDataFormEntityGroup: IEventDataFormEntityGroup[] = [];
+  pageInfo: IPageInfo;
+  total: number;
+  subscriptions: Subscription[] = [];
+  isLoading = false;
+
+  constructor(private edfergService: EventDataFormEntityGroupsService) {}
+
+  ngOnInit() {
```
Comment (Prerna290) [inline apps/shared-components/paid-form-list/paid-form-list.component.ts:23]: remove console

File: apps/commudle-admin/src/app/feature-modules/events/components/event-registrations/form-groups/payment-settings/payment-settings.component.html [hunk 6]
```diff
@@ -63,9 +66,9 @@
                     'com-text-[#8E9BB3]': paidTicketingForm.controls.paid_ticket_setting.get('currency').value === ''
                   }"
                 >
-                  <option value="" selected disabled>Currency</option>
+                  <!-- <option value="" disabled>Currency</option> -->
                   <option value="inr">INR</option>
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/feature-modules/events/components/event-registrations/form-groups/payment-settings/payment-settings.component.html:70]: remove if not required

File: apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-control-panel/community-control-panel.component.html [hunk 7]
```diff
@@ -109,6 +109,41 @@ <h4>{{ community.name }}</h4>
               />
               <span class="title">Hackathons</span>
             </a>
+            <a
+              *ngIf="community.payments_enabled"
+              [routerLink]="['payments']"
+              class="d-flex com-relative"
```
Comment (Prerna290) [inline apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-control-panel/community-control-panel.component.html:115]: move these classes to css file

**Review by arpansac — APPROVED** [review]

**Pull request merged**